spec/Section 5 -- Validation.md

```diff
@@ -198,6 +198,67 @@ query getName {
 }
 ```
 
+### Subscription Operation Definitions
+
+#### Single root field
+
+**Formal Specification**
+
+  * For each subscription operation definition {subscription} in the document
+  * Let {rootFields} be the top level selection set on {subscription}.
+    * {rootFields} must be a set of one.
+
+**Explanatory Text**
+
+Subscription operations must have exactly one root field.
```

Comment:
well, what about the following subscription:

```
subscription {
  post1Comments: newComments(postId: 1) { ... }
  post2Comments: newComments(postId: 2) { ... }
}
```

When post 1 gets a comment, you could probably get a payload like:

```
{ post1Comments: { ... } }
```

And when post 2 gets one, you'd get:

```
{ post2Comments: { ... } }
```

I can't think of any situation where you would get both.

So you're sending two subscription fields in the request, but you're never going to get that entire response. Plus, you can now only unsubscribe to the whole thing at once (maybe a benefit) and it's not clear what to do when one of them has a fatal error - do both get unsubscribed?

From the client's point of view, IMO, having multiple root fields just adds complexity (on a conceptual level) without a lot of benefits.